src/GaussianProcessEmulator.jl

```diff
@@ -472,17 +497,38 @@ function svd_transform(data::Array{FT, 2}, obs_noise_cov::Union{Array{FT, 2}, No
 end
 
 function svd_transform(data::Vector{FT}, obs_noise_cov::Union{Array{FT, 2}, Nothing}) where {FT}
```

Comment:
We can define the svd_transform function outside of the GP constructor, so we don't need a copy of it in here